### PR TITLE
[ZERO TRUST] Updated/Applied Content Strategy

### PR DESCRIPTION
**Pull request by mmalden**

### CODE REVIEW

File: content/cloudflare-one/identity/idp-integration/facebook-login.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ Use these steps to set up Facebook as your identity provider.
 
 1.  Click **Save Changes**.
 
-To test that your connection is working, navigate to **Authentication > Login methods** and click **Test** next to Facebook.
+To test that your connection is working, follow the steps on [SSO Integration](https://developers.cloudflare.com/cloudflare-one/identity/idp-integration/).
```
Comment (kodster28) [inline content/cloudflare-one/identity/idp-integration/facebook-login.md]:
```suggestion
To test that your connection is working, follow the steps on [SSO Integration](/cloudflare-one/identity/idp-integration/#test-idps-on-the-zero-trust-dashboard).
```

Comment (kodster28) [reply content/cloudflare-one/identity/idp-integration/facebook-login.md]: We actually don't have the specify the `developers.cloudflare.com` on our internal links b/c they live within the same folder.

We used to use relative links, but those actually ended up being really tricky to parse and a pain with migration, so now we use absolute links.

File: content/cloudflare-one/policies/browser-isolation/configuration.md [hunk 2]
```diff
@@ -52,12 +52,12 @@ If you would like to temporarily disconnect Browser Isolation, you can do this b
 
 ### macOS
 
-1.  Click on the Cloudflare Logo in the Menu Bar.
-1.  Toggle the blue Connected switch into the Disconnected state.
+1.  Click on the **Cloudflare Logo** in the Menu Bar.
```
Comment (kodster28) [inline content/cloudflare-one/policies/browser-isolation/configuration.md]:
```suggestion
1.  Click the **Cloudflare Logo** in the Menu Bar.
```

File: content/cloudflare-one/policies/browser-isolation/configuration.md [hunk 3]
```diff
@@ -52,12 +52,12 @@ If you would like to temporarily disconnect Browser Isolation, you can do this b
...
+1.  Toggle the blue **Connected** switch into the Disconnected state.
 1.  Refresh your webpage to return to the non-isolated page.
 
 ### Windows
 
-1.  Click on the Cloudflare Logo in the System Tray.
-1.  Toggle the blue Connected switch into the Disconnected state.
+1.  Click on the **Cloudflare Logo** in the System Tray.
```
Comment (kodster28) [inline content/cloudflare-one/policies/browser-isolation/configuration.md]:
```suggestion
1.  Click the **Cloudflare Logo** in the System Tray.
```

**Review by kodster28 — APPROVED** [review]

**Review by ranbel — APPROVED** [review]

**Pull request merged**